nox/_options.py

```diff
@@ -320,6 +327,18 @@ def _session_completer(
         group=options.groups["secondary"],
         help="Re-use existing virtualenvs instead of recreating them.",
     ),
+    _option_set.Option(
+        "R",
```

Comment:
This should be lowercase, right?

Reply:
The option is uppercase (`-R`), so using `r` as the option name would lead to an error:

```python
Traceback (most recent call last):
  File ".../nox/venv/bin/nox", line 33, in <module>
    sys.exit(load_entry_point('nox', 'console_scripts', 'nox')())
  File ".../nox/nox/__main__.py", line 30, in main
    args = _options.options.parse_args()
  File ".../nox/nox/_option_set.py", line 267, in parse_args
    self._finalize_args(args)
  File ".../nox/nox/_option_set.py", line 255, in _finalize_args
    value = getattr(args, option.name)
AttributeError: 'Namespace' object has no attribute 'r'
```

Our `option.name` must match the name that argparse derives for the option. In this case, the option is uppercase `-R` with no equivalent long option, so argparse derives `R` as the option name.

Reply:
Gotcha